openmp-helloworld/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+project(openmp_helloworld)
+
+cmake_minimum_required(VERSION 3.16)
+
+# Search for rocm in common locations
+list(APPEND CMAKE_PREFIX_PATH /opt/rocm/hip /opt/rocm)
+
+# Find HIP
+find_package(hip REQUIRED)
```

Comment:
can we add a comment that user can override AMDGPU_TARGETS in HIP config file?

Reply:
Done